Documentation/kubernetes.md

```diff
@@ -18,24 +18,22 @@ Note that we are striving for even more smooth integration with Kubernetes in th
 
 With your Kubernetes cluster running, Rook can be setup and deployed by simply creating the [rook-operator](/demo/kubernetes/rook-operator.yaml) deployment and creating a [rook cluster](/demo/kubernetes/rook-cluster.yaml).
 
-```
+```bash
 cd demo/kubernetes
 kubectl create -f rook-operator.yaml
-```
 
-This will start the rook-operator pod.  Verify that it is in the `Running` state before proceeding:
-```
+# This will start the rook-operator pod.  Verify that it is in the `Running` state before proceeding:
 kubectl get pod | grep rook-operator
 ```
 
-Now that the rook-operator pod is in the `Running` state, we can create the Rook cluster. See the documentation on [configuring the cluster](cluster-tpr.md).
-```
+Now that the rook-operator pod is running, we can create the Rook cluster. See the documentation on [configuring the cluster](cluster-tpr.md).
```

Comment:
the previous comment has the precise `Running` term, so it seemed more natural as an informal mention of the state